### ISSUE TITLE
New Version

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuza55/opendlp#25]
```
Will you have the newest version in a virtualbox or vmware download soon?
```

Original issue reported on code.google.com by `dvanme...@kc.rr.com` on 1 Aug 2011 at 3:53

**Comment by GoogleCodeExporter** [comment]
```
Yes, I hope to have a new VM released in a few weeks based on the 0.4 release. 
It has been quite a while since the 0.2.2 VM was released in August 2010.
```

Original comment by `andrew.O...@gmail.com` on 1 Aug 2011 at 4:04
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
I just released a new VirtualBox VM based on Ubuntu 11.04 and OpenDLP 0.4.1. 
You can download it here: https://code.google.com/p/opendlp/downloads/list
```

Original comment by `andrew.O...@gmail.com` on 15 Aug 2011 at 6:04
- Changed state: **Fixed**
